projects/plugins/crm/package.json

```diff
@@ -1,29 +1,33 @@
 {
 	"name": "@automattic/jetpack-crm",
-	"version": "5.5.2-alpha",
+	"version": "5.6.0-alpha",
 	"description": "The CRM for WordPress",
 	"author": "Automattic",
 	"license": "GPL-2.0",
 	"scripts": {
-		"start": "npx wp-env start",
-		"stop": "npx wp-env stop",
-		"clean": "npx wp-env clean all"
+		"build": "pnpm run clean && pnpm run build-client",
+		"build-client": "NODE_PATH=\"$PWD/node_modules\" webpack",
+		"clean": "rm -rf build/",
```

Comment:
I suspect I don't need this clean script either as there is no build directory.

Reply:
True. Or you could adjust it to remove the files that actually are built.